projects/components/src/utils/test/datagrid/vcd-datagrid.wo.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { DatagridComponent } from './../../../datagrid/datagrid.component';
+import { WidgetObject } from '../widget-object';
+import { ClrDatagridWidgetObject } from './datagrid.wo';
+
+/**
+ * Widget Object for our VCD DataGrid
+ */
+export class VcdDatagridWidgetObject<R> extends WidgetObject<DatagridComponent<R>> {
+    static tagName = `vcd-datagrid`;
+
+    /**
+     * Gives the header above the grid.
+     */
+    get gridHeader(): string | undefined {
+        return this.findElement('h3') ? this.getText('h3') : undefined;
```

Comment:
Can we instead improve `getText` so that it returns an empty string if a null element is passed to it?